Makefile

```diff
@@ -52,7 +63,7 @@ push: #build ## Push the docker container to the registry & tag latest
 
 VAULTS=`cat devops/vaults.json`
 update-env: ## Update env from 1pass
-	oc -n "$(OPS_REPOSITORY)-tools" exec dc/bcros-cli -- ./scripts/1pass.sh \
+	oc -n "$(OPS_REPOSITORY)-$(TAG_NAME)" exec "dc/vault-service-$(TAG_NAME)" -- ./scripts/1pass.sh \
```

Comment:
Does this get done on every deploy (from GHA CD), or only for main -> dev ?

Reply:
Yes, it will update the envrs from 1pass for all envs.